specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,11 +6,9 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
-The current preview release of Language is 2022-10-01-preview.
+The current preview release of Language is 2023-11-15-preview.
```

Comment:
You need to change line 14 - the `tag` value - to this to actually change the default. It's still 2023-04-01. Doesn't matter whether it's preview or stable. It's more so the latest.